Jenkinsfile

```diff
@@ -33,7 +33,7 @@ pipeline {
       post {
         always {
           junit 'output/**/*.xml'
-          cobertura autoUpdateHealth: true, autoUpdateStability: true, coberturaReportFile: 'coverage.xml', conditionalCoverageTargets: '100, 0, 0', failUnhealthy: true, failUnstable: false, lineCoverageTargets: '100, 0, 0', maxNumberOfBuilds: 0, methodCoverageTargets: '100, 0, 0', onlyStable: false, sourceEncoding: 'ASCII', zoomCoverageChart: false
+          cobertura autoUpdateHealth: true, autoUpdateStability: true, coberturaReportFile: 'coverage.xml', conditionalCoverageTargets: '95, 0, 0', failUnhealthy: true, failUnstable: false, lineCoverageTargets: '95, 0, 0', maxNumberOfBuilds: 0, methodCoverageTargets: '95, 0, 0', onlyStable: false, sourceEncoding: 'ASCII', zoomCoverageChart: false
```

Comment:
where did we get this 95% from?

Reply:
This is a question about quality. I spoke to @eladkug about this one. 100% code coverage is hard to get too even though it is always the goal.
Open to what a good number would be

Reply:
without this change if you will not have 100% code coverage you cant merge your code.
We discussed that we need to make sure we know what we dont cover and improve our selves but not blocking us.

Reply:
Unlike Golang getting 100% coverage in Python is not that difficult with builtins and if the tests actually do cover all the logic paths especially since the code is already at 100%. By reducing this value we are explicitly saying that a lower quality of our product is acceptable and desired which is. Overall I would highly disagree with this change.

Reply:
You are right that we should aim for that 100% and we should eventually get there but we cannot make that a blocker at this stage. We will have leftovers no doubt and they should be addressed when we are less tight for time. This is why I added that TODO from [your pervious comment](https://github.com/cyberark/conjur-api-python3/pull/102#discussion_r532703740) so that when I reach that area, I will address that those tests and improve the percentage